src/term/mod.rs

```diff
@@ -2053,6 +2056,80 @@ mod tests {
 
         assert_eq!(term.grid()[&cursor].c, '▒');
     }
+
+    #[test]
+    fn increase_font_size_works() {
+        let size = SizeInfo {
+            width: 21.0,
+            height: 51.0,
+            cell_width: 3.0,
+            cell_height: 3.0,
+            padding_x: 0.0,
+            padding_y: 0.0,
+        };
+        let config : Config = Default::default();
+        let mut term = Term::new(&config, size);
+        term.change_font_size(10.0);
+
+        let expected_font_size :Size = config.font().size() + Size::new(10.0);
+        assert_eq!(term.font_size, expected_font_size);
+    }
+
+    #[test]
+    fn decrease_font_size_works() {
+        let size = SizeInfo {
+            width: 21.0,
+            height: 51.0,
+            cell_width: 3.0,
+            cell_height: 3.0,
+            padding_x: 0.0,
+            padding_y: 0.0,
+        };
+        let config : Config = Default::default();
+        let mut term = Term::new(&config, size);
+        let negative_font_size_step :f32 = -10.0;
+        term.change_font_size(negative_font_size_step);
+
+        let expected_font_size :Size = config.font().size() + Size::new(negative_font_size_step);
+        assert_eq!(term.font_size, expected_font_size);
+    }
+
+    #[test]
+    fn attempt_do_decrease_font_below_threshold_defaults_to_font_size_step() {
+        let size = SizeInfo {
+            width: 21.0,
+            height: 51.0,
+            cell_width: 3.0,
+            cell_height: 3.0,
+            padding_x: 0.0,
+            padding_y: 0.0,
+        };
+        let config : Config = Default::default();
+        let mut term = Term::new(&config, size);
+
+        term.change_font_size(-100.0);
+
+        let expected_font_size :Size = Size::new(FONT_SIZE_STEP);
+        assert_eq!(term.font_size, expected_font_size);
+    }
+
+    #[test]
+    fn reset_font_size_works() {
+        let size = SizeInfo {
+            width: 21.0,
+            height: 51.0,
+            cell_width: 3.0,
+            cell_height: 3.0,
+            padding_x: 0.0,
+            padding_y: 0.0,
+        };
+        let config : Config = Default::default();
+        let mut term = Term::new(&config, size);
+
+        term.change_font_size(10.0);
+        term.reset_font_size();
+        assert_eq!(term.font_size, term.original_font_size);
```

Comment:
This test is kinda assuming that `term.original_font_size` is actually set properly. It would be better to get the original font size from the config object, it should be in `config.font().size()` or something similar?

Reply:
Done!